src/docs/Components/Accordion/previews/selectable.preview.html

```diff
@@ -0,0 +1,50 @@
+<div class="example">
+  <div class="mc-accordion mc-accordion--s">
+    <input type="checkbox" id="icon" class="mc-accordion__trigger" />
+    <label
+      for="icon"
+      class="mc-accordion__label mc-accordion__label--selectable"
+      aria-expanded="false"
+    >
+      <div class="mc-checkbox mc-accordion__selectable">
+        <input
+          id="checkbox-01"
+          type="checkbox"
+          class="mc-checkbox__input"
+          name="example"
+        />
+      </div>
+      Accordion with checkbox
+    </label>
+    <div class="mc-accordion__content">
+      <p>
+        Fusce iaculis dolor nulla, a maximus ipsum sollicitudin et. Ut
+        condimentum at orci aliquam feugiat. Curabitur sagittis placerat leo sit
+        amet pharetra.
+      </p>
+    </div>
+  </div>
+
+  <div class="mc-accordion ">
+    <input type="checkbox" id="icon" class="mc-accordion__trigger" />
+    <label
+      for="icon"
+      class="mc-accordion__label mc-accordion__label--selectable mc-accordion__label--no-padding"
+      aria-expanded="false"
+    >  
+      <div class="mc-toggle mc-accordion__selectable">
+        <input class="mc-toggle__input" id="toggle" type="checkbox" />
+        <label class="mc-toggle__label" for="toggle">
+        </label>
+      </div>
+      Accordion with toggle
+    </label>
+    <div class="mc-accordion__content">
```

Comment:
@mohamedMok For the checkbox variation, the accordion title must have the checkbox label style 😉